src/core/team-core.js

```diff
@@ -1,24 +1,98 @@
-const {knex} = require('../util/database').connect();
 import _ from 'lodash';
+import BPromise from 'bluebird';
+const {knex} = require('../util/database').connect();
+const redisClient = require('../util/redis').connect().client;
 import {deepChangeKeyCase} from '../util';
 
+
+const KEY_TEAMS = 'teams';
+const KEY_IS_STALE = 'teamsAreStale';
+const KEY_IS_UPDATING = 'teamsAreUpdating';
+const CACHE_TTL = process.env.TEAMS_CACHE_TTL ||  60 * 10; // in 's'
+let init;
```

Comment:
I guess this is a promise that resolves when the module is initialized? Maybe it could be named something like `initializeModulePromise` or similar